todoman/model.py

```diff
@@ -587,6 +589,8 @@ def todos(self, all=False, lists=[], urgent=False, location='',
             params.extend(lists)
         if urgent:
             extra_where.append('AND priority = 9')
+        elif priority:
+            extra_where.append('AND priority = ' + priority)
```

Comment:
Yes I suppose this would be a useful (but not necessary) feature, though @hobarrera only meant that you need to use `<=` instead of `=` here.